src/index.ts

```diff
@@ -21,32 +21,49 @@ export class TestxServer {
   private schema: string;
   private server: Server;
   private serverUrl: string;
-  private dbConnection: Knex;
+  private backendContext: { [id: string]: any };
 
   constructor(schema: string) {
     this.schema = schema;
   }
 
   public async start() {
-    const { server, url } = await this.generateServer();
+    if (!this.backendContext) { await this.generateBackend(); }
+    await this.generateServer();
```

Comment:
Do we really need to regenerate the express app and apollo server between each `stop()` and `start()`, can't we simple make `server.close()` and `server = app.listen(...)` again?

Also we need to reuse the same port between each stop and start, in this case there is a risk that `getAvailablePort()` will not re-pick the same port